src/types.ts

```diff
@@ -102,11 +102,14 @@ export interface Instance {
 
 export interface TokenSale {
   contract: string;
+  bidNonce: string;
   tokenId: string;
   saleAmount: string;
   buyer: string;
   seller: string;
   timestamp: string;
+  paymentToken?: string;
+  topLevelDomainId?: string;
```

Comment:
Left a comment in the subgraph PR, but we shouldn't make this nullable. This is a calculatable value, we need to fill it, either here in the SDK or in the subgraph. (There is a way in the subgraph)